src/jestRunner.ts

```diff
@@ -32,7 +32,8 @@ export class JestRunner {
   // public methods
   //
 
-  public async runCurrentTest(currentTestName?: string) {
+  public async runCurrentTest(currentTestName?: string, updateSnapshots?: boolean) {
```

Comment:
i would prefer i we extend this function the same way it is already done for runCurrentFile
https://github.com/firsttris/vscode-jest-runner/blob/65ec6c3b45a93031d8e4ca55944ccb787d3eb9ec/src/jestRunner.ts#L53

that you have an array of options.

runCurrentTest(currentTestName?: string, options?: string[])

Reply:
Makes sense, I'll update how the params are being passed and remove from context to only have it in command palette.